tests/Microsoft.Data.Analysis.Tests/DataFrameTests.cs

```diff
@@ -802,32 +802,42 @@ public void TestOrderBy()
             df.Columns["Int"][0] = 100;
             df.Columns["Int"][19] = -1;
             df.Columns["Int"][5] = 2000;
+            df.Columns["Int"][6] = null;
```

Comment:
Would it be possible to make a new test that has null values and leave this test untouched?

Reply:
Hi eerhardt, sure thing